anx/provider/loadbalancer/loadbalancer.go

```diff
@@ -379,7 +383,12 @@ func lbStatusFromReconcileStatus(reconStatus map[string][]uint16) *v1.LoadBalanc
 				Protocol: v1.ProtocolTCP,
 			})
 		}
-
+		if hostname := strings.ToLower(service.Annotations[AKEAnnotationHostname]); hostname != "" {
+			ret.Ingress = append(ret.Ingress, v1.LoadBalancerIngress{
+				Hostname: hostname,
+				Ports:    ports,
+			})
```

Comment:
for dualstack services we now create two entries with the same hostname and ports